src/sys/gtl.c

```diff
@@ -772,9 +779,12 @@ void gtlMain(FnBundle* arg0) {
     }
 
     gtlWaitAllGfxTasksDone();
-    while (osRecvMesg(&gtlD_800497E0, NULL, OS_MESG_NOBLOCK) != -1) {}
-    while (osRecvMesg(&gtlResetQueue, NULL, OS_MESG_NOBLOCK) != -1) {}
-    while (osRecvMesg(&gtlGameTickQueue, NULL, OS_MESG_NOBLOCK) != -1) {}
+    while (osRecvMesg(&gtlD_800497E0, NULL, OS_MESG_NOBLOCK) != -1) {
```

Comment:
Maybe consider a `WAIT_UNTIL( x )` for these empty while loops?

Reply:
I think the new formatting of them is fine